feature-pack/src/main/resources/modules/system/layers/base/com/redhat/gss/redhat-support-lib-java/main/module.xml

```diff
@@ -0,0 +1,18 @@
+<module xmlns="urn:jboss:module:1.0" name="com.redhat.gss.redhat-support-lib-java">
+    
+    <resources>
+        <artifact name="${com.redhat.gss:redhat-support-lib-java}"/>
+    </resources>
+
+    <dependencies>
+        <module name="javax.api"/>
+        <module name="javax.ws.rs.api" services="import"/>
+        <!-- <module name="javax.net.ssl" slot='main'/> -->
+        <module name="org.apache.commons.net" slot='main'/>
```

Comment:
I see what you're saying. I tried adding this to the module.xml of redhat-support-lib-java but I couldn't get it to build for some reason. I kept getting the following:

[WARNING] Rule 0: org.apache.maven.plugins.enforcer.BanTransitiveDependencies failed with message:
org.wildfly:wildfly-feature-pack:pom:10.0.0.CR3-SNAPSHOT
   com.redhat.gss:redhat-support-lib-java:jar:2.0.11:compile has transitive dependencies:
      commons-net:commons-net:jar:3.2:compile